src/app/sign-up/sign-up.component.ts

```diff
@@ -11,9 +14,14 @@ import {
 export class SignUpComponent implements OnInit {
   login: string;
   password: string;
-  constructor(private http: HttpClient) {}
+  count$: Observable<string>
+  constructor(private store: Store<{ count: string }>, private http: HttpClient) {
+    this.count$ = store.pipe(select('count'));
```

Comment:
you can initialize `count$` before constructor

Reply:
I think it already has been done, hasn't it?